pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -159,7 +159,7 @@ self: super:
     ];
     configureFlags = attrs.configureFlags or []
       ++ malloc0ReturnsNullCrossFlag
-      ++ lib.optional (stdenv.targetPlatform.useLLVM or false) "ac_cv_path_RAWCPP=cpp";
+      ++ lib.optional (stdenv.cc.isClang && !stdenv.hostPlatform.isDarwin) "ac_cv_path_RAWCPP=cpp";
```

Comment:
i don’t understand what’s going on here. If it’s Clang‐related, we should see it on Darwin.

Reply:
Something is broken with Xorg's configure scripts which only screws up with Clang when there's a target prefix on Linux.